app/helpers/statistics_helper.rb

```diff
@@ -4,13 +4,21 @@
 
 module StatisticsHelper
   def registrations_status statistics
-    if statistics.open_registrations?
+    if !!statistics.open_registrations?
```

Comment:
Add `require "pry"; binding.pry` in the line before that and trigger the code path.